tests/tape_tests/conftest.py

```diff
@@ -118,3 +121,58 @@ def parquet_ensemble_from_hipscat(dask_client):
     )
 
     return ens
+
+
+# pylint: disable=redefined-outer-name
+@pytest.fixture
+def dask_dataframe_ensemble(dask_client):
+    """Create an Ensemble from parquet data."""
+    ens = Ensemble(client=dask_client)
+
+    num_points = 1000
+    all_bands = ["r", "g", "b", "i"]
+    rows = {
+        "id": [8000 + (i % 5) for i in range(num_points)],
+        "time": [float(i) for i in range(num_points)],
+        "flux": [float(i % 4) for i in range(num_points)],
+        "band": [all_bands[i % 4] for i in range(num_points)],
+        "err": [0.1*(i % 10) for i in range(num_points)],
+        "count": [i for i in range(num_points)],
+        "something_else": [None for _ in range(num_points)],
+    }
+    cmap = ColumnMapper(id_col="id", time_col="time", flux_col="flux", err_col="err", band_col="band")
+
+    ens.from_dask_dataframe(
+        dd.from_dict(rows, npartitions=1),
+        column_mapper=cmap,
+    )
+
+    return ens
+
+
+# pylint: disable=redefined-outer-name
+@pytest.fixture
+def pandas_ensemble(dask_client):
+    """Create an Ensemble from parquet data."""
+    ens = Ensemble(client=dask_client)
+
+    num_points = 1000
+    all_bands = ["r", "g", "b", "i"]
+    rows = {
+        "id": [8000 + (i % 5) for i in range(num_points)],
```

Comment:
Maybe not important in tests, but list comprehension is slow, I would prefer to write
```python
8000 + (np.arange(num_points) % 5)
```
or even
```python
8000 + np.tile(np.arange(num_objects), num_points // num_objects)
```
where `num_objects` is 5

Reply:
Thanks for suggesting! Updated the other list comprehensions as well.